qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -126,7 +126,20 @@
             Windows:
                 # Windows guest only works with qxl driver.
                 vga = "qxl"
+        - check_hotplug_virtio_console:
+            only default
+            gagent_check_type = hotplug_virtio_console
+            extra_params += "-device virtio-serial-pci,id=virtio-serial0,bus=pci.0,addr=0x7"
```

Comment:
addr=0x7 is not necessary, I am not sure if it is hard code.  If other devices occupy this addr in future.  Maybe address conflict.